pennylane/templates/subroutines/qsvt.py

```diff
@@ -131,9 +131,7 @@ def qsvt(A, angles, wires, convention=None):
 
         if global_phase:
             with qml.QueuingManager.stop_recording():
-                global_phase_op = qml.exp(
-                    qml.Identity(wires=wires), 0.5j * np.pi * (4 - global_phase)
-                )
+                global_phase_op = qml.GlobalPhase(-0.5 * np.pi * (4 - global_phase), wires=wires)
```

Comment:
`global_phase_op` is only used in line 144 below:
```
if convention == "Wx":
        return qml.prod(global_phase_op, QSVT(UA, projectors))
```

We could just delete these lines and then replace the part below with:
```
if convention == "Wx":
        return qml.prod(qml.GlobalPhase(-0.5 * np.pi * (4 - global_phase), wires=wires), QSVT(UA, projectors))
```
But maybe the original change is better :thinking:

Reply:
The original one is slightly easier to read, but I do not have a strong preference.

Reply:
I prefer the original 👍🏼